### ISSUE TITLE
`Response::interact` doesn't always work

### ISSUE THREAD

**Issue by rustbasic** [emilk/egui#3989]
The Sense recognition function does not work for `Image` objects, so `response.interact(egui::Sense::click())` is used to check for mouse clicks. 
However, after the recent update, left-clicking and right-clicking on an image are recognized about 1 in 100 times.

```rust
        let response = ui.add(image.show_loading_spinner(false));
        let response = response.interact(egui::Sense::click());
        if response.clicked() {
        }
        if response.secondary_clicked() {
        }
```

egui Version: v0.26.0 & master branch

**Comment by cjs-axsh** [comment]
I also have this issue, but have not noticed any sense recognition working on images at all. I use click, drag, and hover on images and none of them are working for me after the update.

**Comment by rustbasic** [comment]
@emilk @cjs-axsh                                                                      
If I use .sense() without using .interact(), it works without any problems.
                                                                                      
```rust                                                                               
        let response = ui.add(image.show_loading_spinner(false).sense(egui::Sense::click()));
        if response.clicked() {                                                       
        }                                                                             
        if response.secondary_clicked() {                                             
        }                                                                             
```

**Comment by cjs-axsh** [comment]
@rustbasic, thanks for the tip. This worked for me in one place where I am interacting with images directly, but did not work for interacting with a frame that contains an image and a label. 

I tested it a little, and it seems that maybe `.interact()` itself is just broken. I tried just commenting out the image in the frame, and was not able to interact with just a frame containing a label either. So maybe unrelated to images? Anyway, I'm still using 0.25.0 for now.
